style.css

```diff
@@ -464,3 +491,991 @@ hr {
   justify-content: center;
   gap: 2.4rem;
 }
+
+@media screen and (min-width: 48em) and (max-width: 60em) {
```

Comment:
Please change all of your media queries to `px` as `em` is a relative measure, not a static one, and it is not a good way to set breaking points for different screen sizes. Additionally, make sure you only use media queries for breaking points starting at 768px following the principles of mobile-first development.